protocol-units/bridge/sources/METHBridge.move

```diff
@@ -0,0 +1,455 @@
+module 0x1::METHBridge {
+    use std::signer;
+    use std::vector;
+    use aptos_framework::coin;
+    use aptos_framework::event;
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_framework::event::{EventHandle};
+    use aptos_framework::account;
+
+    const BRIDGE_ACCOUNT: address = @0x1; //Should change this later to actual bridge address
+    const EPENDING_WITHDRAWAL_NOT_FOUND: u64 = 3;
```

Comment:
Noticed we asserts for `1` and then we have this `const` pointing to `3` .. maybe it would be good to get the two defined here